include/rmm/detail/dynamic_load_runtime.hpp

```diff
@@ -116,6 +121,28 @@ struct async_alloc {
     return runtime_supports_pool and driver_supports_pool;
   }
 
+  /**
+   * @brief Check whether the specified `cudaMemAllocationHandleType` is supported on the present
+   * CUDA driver/runtime version.
+   *
+   * @note This query was introduced in CUDA 11.3 so on CUDA 11.2 this will only return true for
+   * `cudaMemHandleTypeNone`.
+   *
+   * @param handle_type An IPC export handle type to check for support.
+   * @return true if supported
+   * @return false if unsupported
```

Comment:
Shouldn't this just be a single line?
```suggestion
   * @return true if supported, otherwise false
```

Reply:
I wondered the same. Reading the doxygen docs, I saw this and decided it was not an issue:
> Multiple adjacent `\return` commands will be joined into a single paragraph.

https://www.doxygen.nl/manual/commands.html#cmdreturn

Reply:
Yeah I guess that would work then. Joining it explicitly gives us better control over the punctuation and spacing though, so probably best for the purposes of matching our style (and in the same vein as your Cython request, best to be consistent and avoid surprises, even in docs).

Reply:
This is automatically added by the doxygen vscode extension that rapids-compose packages / recommends.

Reply:
You mean the doxygen was autogenerated? That's pretty cool.

I'm not a stickler for this, you wrote our dev docs so you can tell us whether this is compatible with our style guide or not!